app/src/routes/user/user.ctrl.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
+
+const userService = require("../../services/userService");
+
+const output = {
+  login: (req, res) => {
+    res.render("user/login");
+  },
+
+  register: (req, res) => {
+    res.render("user/register");
+  },
+};
+
+const process = {
+  login: async (req, res) => {
+    const userservice = new userService(req.body);
+    const response = await userservice.login();
+
+    const url = {
+      method: "POST",
+      path: "/login",
+      status: response.err ? 400 : 200,
+    };
+    return res.status(url.status).json(response);
```

Comment:
```suggestion
    return res.status(response.status).json(response);
```
서비스에서 status를 리턴해준다면 위에 url 선언은 필요 없을 것 같네요 그리고 다양한 http 상태 코드를 알려줄 수도 있을 것 같구용